release/community-stable/amazonlinux/docker/Dockerfile

```diff
@@ -13,6 +13,9 @@ ARG PS_PACKAGE=powershell-${PS_VERSION}-1.rhel.7.x86_64.rpm
 ARG PS_PACKAGE_URL=https://github.com/PowerShell/PowerShell/releases/download/v${PS_VERSION}/${PS_PACKAGE}
 ARG PS_INSTALL_VERSION=7
 
+# Download the PowerShell Core Linux .rpm and save it
+ADD ${PS_PACKAGE_URL} /tmp/powershell-linux.rpm
```

Comment:
Why did you revert this?  It improved the image size.

Reply:
Actually, I forgotten about the layer size issue since I was helping on some work requirement for standardising syntax. lol.